tekton/pipeline-templates/params/helper/install-list.yml.j2

```diff
@@ -0,0 +1,2 @@
+params:
```

Comment:
This file seems like an unnecessary layer of abstraction compared to whatever is including this including this content directly .. but might make sense as I get further through the changes ...

Reply:
https://ibm-watson-iot.slack.com/archives/C0195MVCEUD/p1675098979911869?thread_ts=1675088015.554689&cid=C0195MVCEUD